cli/go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/kintoproj/kintohub/cli
 
-go 1.16
+go 1.15
```

Comment:
I am not mistaken we are using go 1.13 for everything right now. Let's stick with it.

Reply:
This is what kintoproj/kinto-cli uses. Are you sure?

Reply:
oh ok, then it's fine!

Reply:
There also go 1.12 (kinto-build).